swarm/api/api.go

```diff
@@ -79,12 +101,14 @@ type ErrResolve error
 
 // DNS Resolver
 func (self *Api) Resolve(uri *URI) (storage.Key, error) {
+	apiResolveCount.Inc(1)
 	log.Trace(fmt.Sprintf("Resolving : %v", uri.Addr))
 
 	// if the URI is immutable, check if the address is a hash
 	isHash := hashMatcher.MatchString(uri.Addr)
 	if uri.Immutable() || uri.DeprecatedImmutable() {
 		if !isHash {
+			apiResolveFail.Inc(1)
```

Comment:
I don't think it is correct to log this as a resolve failure, as no resolving was attempted (the user just provided bad input).

Reply:
I finally agree on this one and will remove it